### DISCUSSION TITLE
Mutations Called Serially?

### DISCUSSION THREAD

**Discussion by miles-po** [Netflix/dgs-framework#1140] [General]
According to [the GraphQL spec](https://spec.graphql.org/October2021/#sec-Mutation), mutations are invoked serially rather than in parallel. Does DGS actually enforce this within the engine, invoking the first mutation and its resolvers and waiting for the response before invoking the next one?

**Comment by srinivasankavitha** [comment]
The DGS framework is based on `graphql-java` and relies on that for the core GraphQL processing logic. Per this documentation, this is handled as per the spec: https://www.graphql-java.com/documentation/execution/#asyncserialexecutionstrategy
